kandroid/src/main/kotlin/com/pawegio/kandroid/KApi.kt

```diff
@@ -24,4 +24,12 @@ inline fun toApi(toVersion: Int, inclusive: Boolean = false, action: () -> Unit)
 
 inline fun fromApi(fromVersion: Int, inclusive: Boolean = true, action: () -> Unit) {
     if (Build.VERSION.SDK_INT > fromVersion || (inclusive && Build.VERSION.SDK_INT == fromVersion)) action()
+}
+
+inline fun fromLollipopAbove(action: () -> Unit) {
+    fromApi(Build.VERSION_CODES.LOLLIPOP, true, action)
+}
+
+inline fun fromMarshMellowAbove(action: () -> Unit) {
+    fromApi(Build.VERSION_CODES.M, true, action)
```

Comment:
Yes, it depends on project so better approach is to extract and reuse such functions internally in android app module (when API from specific versions is used more often).